.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Where did this file come from? There may be a case for adding this but we could discuss that on a separate issue / pull request.

Reply:
This is just to install the hooks. I will remove this file.